components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/utils/RestApiAdminUtils.java

```diff
@@ -74,6 +77,38 @@ public static boolean isBlockConditionAccessibleToUser(String user, BlockConditi
                 .equals(userTenantDomain);
     }
 
+    /**
+     * Validate the required properties of Custom Rule Policy
+     *
+     * @param customRuleDTO custom rule object to check
+     * @param httpMethod    HTTP method of the request
+     */
+    public static void validateCustomRuleRequiredProperties(CustomRuleDTO customRuleDTO, String httpMethod)
+            throws APIManagementException {
+
+        String propertyName;
+
```

Comment:
Shall we remove the empty lines

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/8639/commits/b64a60e14586ec830be15406b5eefceb791fbbc1